### PR TITLE
chore: Remove cpan

### PR DESCRIPTION
**Pull request by Hoverbear**
Remove our dependency on cpan rename. (Please let this test on CI!)

### CODE REVIEW

**Review by jamtur01 — APPROVED** [review]
Assuming this passes - hellz yes.

**Comment by Hoverbear** [comment]
Yeah I wasn't able to test on windows yet, but I just set up a WSL instance and I'm testing now. :)

**Comment by Hoverbear** [comment]
Okay great this works for me locally now.

File: scripts/package-deb.sh [hunk 1]
```diff
@@ -74,8 +74,11 @@ cargo deb --target "$TARGET" --deb-version "$PACKAGE_VERSION" --no-build
 
 # Rename the resulting .deb file to use - instead of _ since this
 # is consistent with our package naming scheme.
-# shellcheck disable=SC2016
-rename -v 's/vector_([^_]*)_(.*)\.deb/vector-$2\.deb/' "target/$TARGET/debian"/*.deb
+for file in target/"${TARGET}"/debian/*.deb; do
+  base=$(basename "${file}")
+  renamed=${base//_/-}
```
Comment (MOZGIII) [inline scripts/package-deb.sh:79]: This looks a bit off - the original had `([^_]*)_(.*)` into `$2`, cutting the first part (version) in addition to switching from `_` to `-`.

Comment (Hoverbear) [reply scripts/package-deb.sh:79]: Yeah, you're right, thank you. :)

**Pull request merged**